packages/vkui/src/components/ConfigProvider/ConfigProvider.tsx

```diff
@@ -37,6 +50,31 @@ export const ConfigProvider = (props: ConfigProviderProps) => {
     ...props,
   };
 
+  // TODO [>=6]: Удалить данный бэкпорт
+  const hasCustomPanelHeaderAfter =
+    props.webviewType && props.hasCustomPanelHeaderAfter === undefined
+      ? props.webviewType === WebviewType.VKAPPS
+      : hasCustomPanelHeaderAfterMerged;
+
+  if (process.env.NODE_ENVIRONMENT === 'development') {
+    // TODO [>=6]: удалить warn
+    let webviewTypeRule = '';
+    if (props.webviewType) {
+      webviewTypeRule =
+        props.webviewType === WebviewType.INTERNAL
+          ? 'замените webviewType={WebviewType.INTERNAL} на hasCustomPanelHeaderAfterProp={false}'
+          : 'замените webviewType={WebviewType.VKAPPS} на hasCustomPanelHeaderAfterProp={true}';
+    }
+    warn(`[@vkontakte/vk-bridge's deprecated] Если вы используете VK Bridge, то:
+1. используйте хук \`useAppearance()\` из @vkontakte/vk-bridge-react и результат передайте в параметр \`appearance\`;
+2. передайте \`bridge.isWebView()\` в параметр \`isWebView\`;
+${webviewTypeRule}
+
+Подробности см. https://github.com/VKCOM/VKUI/issues/5049
```

Comment:
```suggestion
    warn(`[@vkontakte/vk-bridge's deprecated] Если вы используете VK Bridge, то:
1. используйте хук useAppearance() из @vkontakte/vk-bridge-react и результат передайте в параметр appearance;
2. передайте bridge.isWebView() в параметр isWebView;
${webviewTypeRule}

Подробности см. https://github.com/VKCOM/VKUI/issues/5049
```
Я бы удалила кавычки, потому что консоль у нас все равно не рендерит это красиво.

p.s. (Не в рамках этого PR, естественно) Функцию, что ли, вокруг warn на эту тему написать, чтобы у нас были самые стильные варнинги на районе...